apps/examples/wifi_manager_sample/wm_test.c

```diff
@@ -412,13 +412,13 @@ void wm_set_info(void *arg)
 	WM_TEST_LOG_START;
 	struct options *ap_info = (struct options *)arg;
 	wifi_manager_ap_config_s apconfig;
-	strncpy(apconfig.ssid, ap_info->ssid, WIFIMGR_SSID_LEN-1);
+	strncpy(apconfig.ssid, ap_info->ssid, WIFIMGR_SSID_LEN);
 	apconfig.ssid_length = strlen(ap_info->ssid);
-	apconfig.ssid[WIFIMGR_SSID_LEN-1] = '\0';
+	apconfig.ssid[WIFIMGR_SSID_LEN] = '\0';
```

Comment:
A buffer overflow occurs. WIFIMGR_SSID_LEN-1 is correct

Reply:
The size of (apconfig.)ssid is WIFIMGR_SSID_LEN + 1. Thanks for the comment.